src/components/profile/edit/EditPersonalInfo.vue

```diff
@@ -150,33 +154,35 @@
         :profileFieldObject="alternativeName"
       />
 
-      <hr role="presentation" />
-
-      <div class="edit-personal-info__label">
-        <label for="field-official-job-title">Official Job Title </label
-        ><Tooltip
-          buttonText="Open official job title info"
-          alternateButtonText="Close official job title info"
-          >This is your job title as provided by our HR Team (Workday). Staff
-          information is not editable and is shared with all Staff and NDA’d
-          Mozillians. Please contact
-          <a href="mailto:peopleops@mozilla.com">peopleops@mozilla.com</a>
-          to request a change to your job title.</Tooltip
-        >
-      </div>
-      <input
-        type="text"
-        id="field-official-job-title"
-        disabled
-        :value="staffInformation.title.value"
-      />
-      <PrivacySetting
-        label="Title privacy levels"
-        id="field-title-privacy"
-        profileFieldName="staffInformation.title"
-        :profileFieldObject="staffInformation.title"
-        :disabled="true"
-      />
+      <template v-if="scope.isStaff">
```

Comment:
Can we take this moment to refactor this component? We are duplicating a lot of markup here. The fact that we keep having to add ```v-if="scope.isStaff"``` to html that looks almost identical seems like a large indicator we should look at refactoring some of this code.